docs/en/tutorials/hook.md

```diff
@@ -129,6 +130,14 @@ The four features mentioned above are described below.
   default_hooks = dict(checkpoint=dict(type='CheckpointHook', interval=1, save_best='accuracy', rule='less', published_keys=['meta', 'state_dict']))
   ```
 
+- Control the epoch number or iteration number at which checkpoint saving begins
+
+  If you want to set the number of epochs or iterations to control the start of saving weights, you can set the `save_begin` parameter, defaults to 1, which means saving checkpoints from the beginning of training. For example, if you train for a total of 10 epochs, and `save_begin` is set to 6, then the checkpoints for epochs 6, 7, 8, 9, and 10 will be saved. If `interval=2`, only save checkpoints for epochs 6, 8 and 10.
```

Comment:
if `save_begin` is set to 5, and the interval is 2, which epochs should be saved?

Reply:
5, 7, 9
or
6, 8, 10

Reply:
Suggest setting `save_begin` as 0 by default. Here is an example of saving checkpoints.

interval=2
- save_begin=0: 2, 4, 6, 8, 10   keep backward compatible.
- save_begin=1: 1, 3, 5, 7, 9
- save_begin=5: 5, 7, 9

Reply:
oki, i will change my code

Reply:
Already fix it.